### ISSUE TITLE
Cannot deploy application.

### ISSUE THREAD

**Issue by carla2z** [stackblitz/bolt.new#614]
### Describe the bug

I was able to deploy my first project To netlify.com. After doing that, I cannot deploy any more projects to the service. I get the following 
An unknown error occurred while running an action.

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-ionsdl

### Steps to reproduce

Click deploy or ask for to to deploy in the chat. I still get the same error.

### Expected behavior

the app would deploy. 

### Platform

```
Browser name = Chrome
Full version = 129.0.0.0
Major version = 129
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36 Edg/129.0.0.0
performance.memory = {
  "totalJSHeapSize": 129591299,
  "usedJSHeapSize": 105313287,
  "jsHeapSizeLimit": 4294705152
}
Username = carla2z
Chat ID = eb518dd7726c
```

### Additional context

Error:
```
An unknown error occured while running an action.
```

**Comment by carla2z** [comment]
I've initiated the deployment process to Netlify. The build command npm run build will be used to create a production-ready version of your application, and the output will be stored in the dist directory.

Now, we'll wait for the deployment to complete. Once it's done, I'll provide you with the deployment status and the URL where your site is live.

An error occured while streaming this message.

**Comment by kc0tlh** [comment]
Thanks for the report and patience during this beta period! Tracking all deployment related bug fixes in issue #499
